docs/reference/datatiers.asciidoc

```diff
@@ -2,40 +2,66 @@
 [[data-tiers]]
 == Data tiers
 
-A _data tier_ is a collection of nodes with the same data role that
-typically share the same hardware profile:
-
-* <<content-tier, Content tier>> nodes handle the indexing and query load for content such as a product catalog.
-* <<hot-tier, Hot tier>> nodes handle the indexing load for time series data such as logs or metrics
-and hold your most recent, most-frequently-accessed data.
-* <<warm-tier, Warm tier>> nodes hold time series data that is accessed less-frequently
+A _data tier_ is a collection of <<modules-node,nodes>> within a cluster which share the same 
+<<node-roles,data node role>>. Elastic recommends this collection of nodes also shares the same 
+hardware profile to avoid <<hotspotting,hot spotting>>. Data tiers' usage generally splits along 
+<<data-management,data categories>> for _content_ and _time series_ data. {es} available 
+data tiers:
+
+* <<content-tier,Content tier>> nodes handle the indexing and query load for content 
+indices, such as a <<system-indices,system index>> or a product catalog.
```

Comment:
This might be another 😕 point for me then if we can discuss:

1. Lower down on the existing page under [Content header](https://www.elastic.co/guide/en/elasticsearch/reference/master/data-tiers.html#content-tier) already says "System indices and other indices that aren’t part of a data stream are automatically allocated to the content tier." which is why I didn't realize I might be misunderstanding. 
1. Support encourages users to keep all system indices on hot/content. Does Dev agree?
2. AFAIK (and it's an ongoing discussion / definition-problem) system indices are the indices which report from the snapshot's feature states. So from the [unofficial list I wrote for Support](https://github.com/elastic/elasticsearch/issues/50251#issuecomment-1107637953) we later learned e.g. `.ilm-history` and `.kibana-event-log` don't qualify as system indices. So e.g. only (A) qualify as system indices and AFAICT that subset doesn't have time series data (at least no indices which'd rollover). 

<details><summary>(A)</summary>

---

```
{
  "feature_states": [
    {
      "feature_name": "security",
      "indices": [".security-tokens-7",".security-7",".security-profile-8"]
    },
    {
      "feature_name": "geoip",
      "indices": [".geoip_databases"]
    },
    {
      "feature_name": "async_search",
      "indices": [".async-search"]
    },
    {
      "feature_name": "machine_learning",
      "indices": [".ml-inference-native-000002",".ml-inference-000005",".ml-config"]
    },
    {
      "feature_name": "transform",
      "indices": [".transform-internal-007"]
    },
    {
      "feature_name": "kibana",
      "indices": [
        ".kibana_analytics_8.12.2_001",
        ".kibana_task_manager_8.12.2_001",
        ".kibana_ingest_8.12.2_001",
        ".apm-custom-link",
        ".apm-agent-configuration",
        ".kibana_8.12.2_001",
        ".kibana_security_session_1",
        ".kibana_security_solution_8.12.2_001",
        ".kibana_alerting_cases_8.12.2_001"
      ]
    },
    {
      "feature_name": "tasks",
      "indices": [".tasks"]
    },
    {
      "feature_name": "fleet",
      "indices": [
        ".fleet-agents-7",
        ".fleet-enrollment-api-keys-7",
        ".fleet-actions-7",
        ".fleet-policies-7",
        ".fleet-servers-7",
        ".fleet-policies-leader-7"
      ]
    }
  ]
}
```

---

</details>

Reply:
> Support encourages users to keep all system indices on hot/content.

Users cannot control these at all. System indices cannot be configured apart from specialized APIs. Generally, we shouldn't be talking about system indices with users (if at all), since they are meant to be used only for system usage.

Reply:
Is that a recent change? Because users sending `.kibana*` or `.reporting*` system indices or or `.alert*` if they count as system indices to warm/cold tier is an ongoing lowkey concern.

Reply:
What is the concern with them moving the indices?

Inspecting the code here, we differentiate between internal and external system indices, and those managed by ES and unmanaged by ES (I stand corrected, sorry about my confusion here). It looks like we do have a concept of a system index that a user could influence somewhat (through setting an appropriate origin).

Reply:
> What is the concern with them moving the indices?

Specifically thinking of the SIEM use-case, moving `.alert*` to cold can cause Rules looking over it to timeout which led towards [kibana#172162](https://github.com/elastic/kibana/pull/172162). 

But overall where we generally think of hot as most performant of hardware to keep Kibana as performant as possible we keep it in hot. Is your understanding that `.kibana*` should be able to be hosted anywhere aside from frozen without causing performance issues?

Reply:
FWIW, I'm grateful for the discussion as this has been very helpful and informative, but for the literal line change requested, I'm convinced now we need to standardize a system index definition before including it as an example, cheers. 

FYI in case you're interested, Support discussion in [support-tech-lead#1351](https://github.com/elastic/support-tech-lead/issues/1351).